no_op.md

```diff
@@ -0,0 +1,89 @@
+# No Op
```

Comment:
For testing this particular pipeline, I think surfacing any accession codes that we use for "real samples" (if applicable) could be helpful. Submitter-processed data can be all over the place!

Reply:
I'm not sure if I 100% understood this, but I added some additional info to the end of the Future Tests section of No Op. LMK if I am on the right track.